grapple/urls.py

```diff
@@ -23,7 +15,7 @@ def graphiql(request):
         "supports_subscriptions": has_channels,
     }
     if has_channels:
-        graphiql_settings["SUBSCRIPTIONS_TRANSPORT_VERSION"] = "0.9.19"
+        graphiql_settings["SUBSCRIPTIONS_TRANSPORT_VERSION"] = "0.8.3"
```

Comment:
I had to downgrade the version of this script because it lead to errors in graphiql

Reply:
worth adding a comment/todo there to that effect